### DISCUSSION TITLE
VNC Remote after OS Installs using PINN on Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by ericmagidson** [procount/pinn#652] [Q&A]
First, Thank you for all this great information on using PINN on Pi

I was able to configure and use VNC for the initial boot and choose the OS (multiple) I wanted to install.  After that though, I can no longer seem to get in via VNC, even though I have set a startup timeout in the recovery file.  The screen on the pi states that it is set for VNC but no connection is made.

I would just like it to come up with the PINN menu so that I can choose an OS to boot,  I can do this on Bootberry but can't figure out what I am missing on PINN.

Please let me know if you need more information and again, thank you so very much.

**Comment by procount** [comment]
On the initial PINN boot, there is no OS installed, so it will wait in the Recovery menu.
After you have installed OSes, PINN will boot into the last booted OS after a timeout. Normally this is too quick for you to interrupt it using VNC.
1. I always suggest to use `vncshare` instead of `vncinstall`, so that you can still see what is going on on your local monitor.
2. Which timeout have you set?
3. Add `forcetrigger` to recovery.cmdline and ensure you have used `remotetimeout=30` (as an example timeout)
